4-WebApp-your-API/4-1-MyOrg/AppCreationScripts/Configure.ps1

```diff
@@ -174,191 +171,185 @@ Function CreateAppRole([string] $types, [string] $name, [string] $description)
     return $appRole
 }
 
+# Pre-requisites
+if ($null -eq (Get-Module -ListAvailable -Name "Microsoft.Graph.Applications")) {
+    Install-Module "Microsoft.Graph.Applications" -Scope CurrentUser 
+}
+
+Import-Module Microsoft.Graph.Applications
```

Comment:
idelly i'd place it outside any of the methods, like below, where the check and import for AzureAdModule was

Reply:
moved to the end of the script